cloud-native-runtimes/how-to-guides/troubleshooting.hbs.md

```diff
@@ -2,6 +2,30 @@
 
 This topic tells you how to troubleshoot Cloud Native Runtimes, commonly known as CNRs, installation, or configuration.
 
+## <a id='updates-fail'></a> Updates fail with annotation is immutable
+
+### Symptom
+
+After creating a web workload in TAP 1.6.3 or below, then upgrading beyond TAP 1.6.3, and attempting to update your web workload, you will see the following error
+
+```
+API server says: admission webhook "validation.webhook.serving.knative.dev" denied the request: validation failed: annotation value is immutable: metadata.annotations.serving.knative.dev/creator (reason: BadRequest)
+```
+
+### Explanation
+
+Knative adds annotations to Knative Services. Kapp Controller, which is the orchestrator underneath workloads, does validation that created objects are not modified by others. This conflict between Kapp Controller and Knative is a known and expected behaviour but is mitigated by a configuration passed into Kapp Controller. This configuration specifies that the annotations Knative adds are ok and thus should not fail the validation. This configuration is added dynamically in the delivery supply chain.
```

Comment:
nit:

you refer to it as "a configuration passed into Kapp Controller". Personally, I don't think you need to allude to it like this. Rather, I think you can just call it a "kapp Config" ([docs](https://carvel.dev/kapp/docs/v0.45.0/config/#overview)).

and then "this configuration.." could become "the kapp Config" or "the kapp Config file".